virt/virt-2-5-release-notes.adoc

```diff
@@ -9,14 +9,14 @@ toc::[]
 Red Hat {VirtProductName} {VirtVersion} is supported for use on {product-title} 4.6 clusters. {VirtProductName} enables you to bring traditional virtual machines (VMs) into {product-title} where they run alongside containers, and are managed as native Kubernetes objects.
 
 //Branding-Logo
-{VirtProductName} is represented by the image:Operator_Icon-OpenShift_Virtualization-5.png[{VirtProductName}] logo.
+{VirtProductName} is represented by the image:Operator_Icon-OpenShift_Virtualization-5.png[{VirtProductName},40,40] logo.
```

Comment:
This logo seemed really big in the original.

Before:

![Screenshot from 2020-11-10 09-45-12](https://user-images.githubusercontent.com/3442316/98704174-9253b700-2339-11eb-82cd-6b1003e87811.png)

After:

![Screenshot from 2020-11-10 09-45-23](https://user-images.githubusercontent.com/3442316/98704194-997ac500-2339-11eb-89bb-ce5606cf51d6.png)